modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/search_bar_configuration_suggestions/contributor_input_set_item/Commerce.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import React from 'react';
+
+import InputSetItemHeader from './InputSetItemHeader';
+import CharacterThresholdInput from './inputs/CharacterThresholdInput';
+import DisplayGroupNameInput from './inputs/DisplayGroupNameInput';
+import SizeInput from './inputs/SizeInput';
+
+function Commerce({index, onBlur, onInputSetItemChange, touched, value}) {
```

Comment:
why do we need this? can't we just use the basic contributor? this looks like the basic one

Reply:
Hi @andrea-ale-sbarra , 

Thanks for taking a look! The reason I didn't use the js for the basic contributor is that there's lines that dictate the name and description of the contributor: https://github.com/liferay-commerce/liferay-portal/pull/4784/files/cac11895a352c5b68be639eef8a1b169176ceef4..83dc01a810d44acfc0a083e90ba229160d021de4#diff-4fab32465b7647e9c245d4cc1de2d3c8e5a0eaca03f3d218bbfca7cbb1e30209R25-R33

Since this is a separate contributor from the Basic one, we still need this so it's properly listed.

I'll work on combining the commit history.

Reply:
I've combined the commits and repushed them.

Reply:
I see, I just wonder if there's a chance to put this js contributor in some of our modules, if it's not possible let's keep it like this.

Reply:
ehi @wanderlast why don't we change the basic component, making it accepting these 2 labels as parameter this way we won't need to maintain a new componente which is just a copy of the basic one